regulations/build/regulations/history/official/2019-05-01/index.html.erb

```diff
@@ -0,0 +1,883 @@
+<% provide(:title, 'WCA Regulations') %>
+<div class="container"><h1>WCA Regulations</h1>
+<div class="version">Version: May 1, 2019<br/>[<code><a href="https://github.com/thewca/wca-regulations/tree/official/">official</a>:<a href="https://github.com/thewca/wca-regulations/commits/unknown">unknown</a></code>]</div>
```

Comment:
See my comment on the guidelines html, as it also applies here :)

Reply:
```suggestion
<div class="version">Version: May 1, 2019<br/><b>This is NOT the current official version</b><br/>[<code><a href="https://github.com/thewca/wca-regulations/tree/official/">official</a>:<a href="https://github.com/thewca/wca-regulations/commits/72bccb9">72bccb9</a></code>]</div>
```